src/xdp/trn_transit_xdp.c

```diff
@@ -792,6 +793,22 @@ int _transit(struct xdp_md *ctx)
 
 	int action = trn_process_eth(&pkt);
 
+	/* 
+	 * Look up the entry in the metrics table, the key used is "0" 
+	 * This key need to keep constant with the key used in user prog.
+	 */
+	__u32 metrics_key = 0; 	
+	pkt.rec = bpf_map_lookup_elem(&metrics_table, &metrics_key);
```

Comment:
@phudtran Hi Phu, which variable can be used as the unique id of a host here

Reply:
I'm not aware of a pre-existing ID unique to each node that you can use here.
Perhaps you can push down the hostname of the node and use it as the key here.

Reply:
Thanks, it is fixed. I use host ip as the unique key per host